hack/components/bump-multus.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+source hack/components/yaml-utils.sh
+source hack/components/git-utils.sh
+
+echo 'Bumping multus'
+MULTUS_URL=$(yaml-utils::get_component_url multus)
+MULTUS_COMMIT=$(yaml-utils::get_component_commit multus)
+MULTUS_REPO=$(yaml-utils::get_component_repo ${MULTUS_URL})
+
+TEMP_DIR=$(git-utils::create_temp_path multus)
+trap "rm -rf ${TEMP_DIR}" EXIT
+MULTUS_PATH=${TEMP_DIR}/${MULTUS_REPO}
+
+echo 'Fetch multus sources'
+git-utils::fetch_component ${MULTUS_PATH} ${MULTUS_URL} ${MULTUS_COMMIT}
+
+
+echo 'Split image yamls and choose only needed objects'
+(
+	cd ${MULTUS_PATH}
+	mkdir -p config/cnao
+	cp images/multus-daemonset.yml config/cnao
+
+	$(yaml-utils::split_yaml_by_seperator config/cnao multus-daemonset.yml)
+	rm config/cnao/multus-daemonset.yml
+
+	touch config/cnao/001-multus.yaml
+	for f in config/cnao/*
+	do
+		if [ -z "$(cat $f | grep kube-multus-ds-ppc64le)" ]; then
+			cat $f >> config/cnao/001-multus.yaml
+		fi
+	done
+)
+
+echo 'parametize and copy manifests'
+rm -rf data/multus/*
+(
+	cat ${MULTUS_PATH}/config/cnao/001-multus.yaml |
+		sed 's/kube-system/{{ .Namespace }}/' |
+		sed 's/image: .*/image: {{ .MultusImage }}\n        imagePullPolicy: {{ .ImagePullPolicy }}/' |
```

Comment:
Could we use yq or something proper to parse the YAML and replace values? This would silently break if there is a change in the original YAML.